src/lib/mousetrap-record.js

```diff
@@ -0,0 +1,225 @@
+/**
+ * This extension allows you to record a sequence using Mousetrap.
```

Comment:
My concern was not about the location of the file. I was thinking about including Mousetrap through bower.json and either contributing the fixes to the upstream Mousetrap either maintaining our own fork with the fixes/enhancement. 

Take into account that currently Mousetrap lives here https://github.com/mousetrap-js/mousetrap (not in the original ccampbell repo). They look very welcoming to fixes and improvements.

So in short, don't copy&paste&modify code but collaborate with the maintainers.